pl_examples/The_BoringModel.ipynb

```diff
@@ -0,0 +1,344 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "The BoringModel.ipynb",
```

Comment:
You can also update the default issue link here: it's in

https://github.com/PyTorchLightning/pytorch-lightning/blame/master/.github/ISSUE_TEMPLATE/bug_report.md#L19

Reply:
Hi! 

I updated the link with a path to the master path of the notebook. The link wont work as there is not a notebook there yet. We could merge like this or leave it to a second PR and its properly tested